Documentation/how-to-review-a-pr.md

```diff
@@ -0,0 +1,70 @@
+# PR Review Policy
+
+To ensure code quality, maintainability, and stability, all Pull Requests (PRs) follow a three-step review process. This process includes automated checks followed by two distinct types of human review.
+
+## Automated: 1st Step
+Before any human review, all PRs must successfully pass a series of automated checks. These tools provide the first line of defense against common issues.
+
+### Linter & Formatter
+Automatically fixes all style and formatting issues.
+
+We enforce a shared style guide (for example, Mantra will reuse the style guide from nt-gui).
```

Comment:
```suggestion
Where appropriate, we enforce a shared style guide (for example, Mantra will reuse the style guide from nt-gui, Python tests generally formatted with black across all our projects). Some projects on the same language do not share the style for historical reasons (prime example C/C++ from mender not applied to mender-mcu)
```

Reply:
Maybe we don't need to extend so munch on the examples, but at least I wanted to add some room here for projects that do not share style across the same programming language.